src-shared/rest-api.js

```diff
@@ -34,7 +34,7 @@ const uri = {
   deletePackageNotification: 'deletePackageNotification',
   duplicateEndpoint: '/duplicateEndpoint',
   duplicateEndpointType: '/duplicateEndpointType',
-  dirtyFlag: '/dirtyFlag',
```

Comment:
Why has the previous API gotten removed?

Reply:
adding back. I thought it was not used but it is in the studio-rest-api files..